### PR TITLE
Remove redundant arguments

### PR DESCRIPTION
**Pull request by PercyDan54**

### CODE REVIEW

File: osu.Game/Screens/Play/HUD/LegacyComboCounter.cs [hunk 1]
```diff
@@ -170,9 +170,9 @@ private void transformPopOut(int newValue)
             popOutCount.FadeTo(0.75f);
             popOutCount.MoveTo(Vector2.Zero);
 
-            popOutCount.ScaleTo(1, pop_out_duration, pop_out_easing);
-            popOutCount.FadeOut(pop_out_duration, pop_out_easing);
-            popOutCount.MoveTo(displayedCountSpriteText.Position, pop_out_duration, pop_out_easing);
+            popOutCount.ScaleTo(1, pop_out_duration);
+            popOutCount.FadeOut(pop_out_duration);
+            popOutCount.MoveTo(displayedCountSpriteText.Position, pop_out_duration);
```
Comment (PercyDan54) [inline osu.Game/Screens/Play/HUD/LegacyComboCounter.cs:175]: Introduces a new unused const.
Shall I revert this

Comment (peppy) [reply osu.Game/Screens/Play/HUD/LegacyComboCounter.cs:175]: Removing the const here is fine. It can be added back if it's ever required.

**Comment by PercyDan54** [comment]
I'm leaving this for tomorrow

**Comment by peppy** [comment]
If we're in a good state project wide, may be a good time to turn the rider inspection up to "suggestion" or "warning".

**Comment by bdach** [comment]
Ran inspectcode locally and it looked to be fine.

I've made it a warning mostly to prevent this sort of PR being submitted in the future (violations will just fail CI now). That, and there's only one inspection at the suggestion level in the entire dotsettings file.

**Review by peppy — APPROVED** [review]

**Pull request merged**